doc/src/build/wallet.md

```diff
@@ -349,16 +349,18 @@ FLAGS:
 
 OPTIONS:
         --gas <gas>                ID of the gas object for gas payment, in 20 bytes Hex string
+        --gas-budget <gas-budget>    Gas budget for this transfer
         --object-id <object-id>    Object to transfer, in 20 bytes Hex string
         --to <to>                  Recipient address
 ```
 To transfer an object to a recipient, you will need the recipient's address,
 the object ID of the object that you want to transfer,
-and the gas object ID for the transaction fee payment.
+and the gas object ID for the transaction fee payment. Gas budget sets a cap for how much gas you want to spend.
+We are still finalizing our gas metering mechanisms. For now, just set something large enough.
```

Comment:
Can we suggest an amount? "...enough, for example..."

Reply:
Nevermind. I see 1000 in the example below.

Reply:
We should probably have a default value if not specified.